src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.SolutionChecksumUpdater.cs

```diff
@@ -119,11 +119,36 @@ private void EnqueueChecksumUpdate()
                 _event.Release();
             }
 
-            private Task SynchronizePrimaryWorkspaceAsync(CancellationToken cancellationToken)
+            private async Task SynchronizePrimaryWorkspaceAsync(CancellationToken cancellationToken)
             {
-                return _service.Workspace.SynchronizePrimaryWorkspaceAsync(_service.Workspace.CurrentSolution, cancellationToken);
+                var workspace = _service.Workspace;
```

Comment:
this is just a move.  i didn't change the logic.

Note: if you look above https://github.com/dotnet/roslyn/pull/42744/files#diff-312ddd3b1f5afe7766ce7c2d741d9370R93 you'll see that our trigger for doing this is any workspace change.  So we don't need a redundant synchronizer that only runs for new solutions.